pkg/controller/operators/olm/operator.go

```diff
@@ -1888,10 +1888,17 @@ func (a *Operator) transitionCSVState(in v1alpha1.ClusterServiceVersion) (out *v
 			return
 		}
 
-		if out.HasCAResources() {
+		// Only update certificate status if:
+		//  - there the CSV has CAResources; and
+		//  - it has not already been set, or they should be rotated
+		// See bug: https://bugzilla.redhat.com/show_bug.cgi?id=2020484 for more information
```

Comment:
```suggestion
		// Only update certificate status if:
		//  - the CSV has CAResources; and
		//  - it has not already been set, or they should be rotated
```

Reply:
+1 IRT dropping the BZ reference.

Reply:
BZ is captured in the Pull Request description anyways.

Reply:
Lol - just wrote that in the previous response. I'm good with removing that.